tools/clang/test/HLSLFileCheck/validation/callgraph/deriv-in-nested-fn-cs-lib65.hlsl

```diff
@@ -0,0 +1,55 @@
+// RUN: %dxilver 1.8 | %dxc -T lib_6_5 -Vd %s | %D3DReflect %s | %FileCheck %s -check-prefixes=RDAT
+
```

Comment:
Again, should validation be disabled here? 

```suggestion

// Verifies that use of a group barrier in a nested call in a cs library shader
// produces no diagnostics and serves as the source for like-named .ll files

```

Reply:
Yes, validation needs to be disabled so we can produce the RDAT and check that the flags were propagated as expected.  I'll add a comment.